build.sh

```diff
@@ -24,6 +24,10 @@ systemdsystempresetdir=$(pkg-config systemd --variable=systemdsystempresetdir)
 subagentdir=$prefix/subagents
 
 . VERSION
+BUILD_INFO_IMPORT_PATH="github.com/GoogleCloudPlatform/ops-agent/version"
+BUILD_X1="-X ${BUILD_INFO_IMPORT_PATH}.BuildDistro=$(lsb_release -is | tr A-Z a-z)$(lsb_release -rs)"
+BUILD_X2="-X ${BUILD_INFO_IMPORT_PATH}.Version=${PKG_VERSION}"
```

Comment:
This should be available as one string in the `${DEB_VERSION}` variable in `debian/rules` and `%{version}-%{release}` in the `.spec` file.

Reply:
See my other comment. UserAgent format has been defined after rounds of discussions: b/170441793#comment15. I understand that you think for debian `version~build_diistro` shall be the final look as a whole and for rpm packages `version-release` shall be it.

@qingling128  on deciding if we want to have ~ for APT package and - for rpm pacakge

Reply:
I took that comment as a general example, not a literal version number. But if you want to be pedantic, that comment says `1.0.3-stretch` but your code here is going to generate `1.0.3-debian9`, so you're already not following the format in the comment.

Reply:
My neglect, that can just be changed to:
```sophieyfang_google_com@debian9-ansible-test:~$ lsb_release -c
Codename:	stretch
```
Besides that, what else you think the tradeoff is?

Reply:
[sophieyfang_google_com@centos8-5 ~]$ lsb_release -a
LSB Version:	:core-4.1-amd64:core-4.1-noarch
Distributor ID:	CentOS
Description:	CentOS Linux release 8.3.2011
Release:	8.3.2011
Codename:	n/a

sophieyfang_google_com@debian9-ansible-test:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 9.13 (stretch)
Release:	9.13
Codename:	stretch

ummmmm... @qingling128 , for those distros that have codename shall we use code name, for those that doesn't have it shall be use Release just as it is or transformed it to el8?

Reply:
Google-Cloud-Ops-Agent-Logging/1.0.5 (BuildDistro=stretch;Platform=Linux;ShortName=Debian;ShortVersion=9 is the new format

Google-Cloud-Ops-Agent-Logging/1.0.5 (BuildDistro=9;Platform=Linux;ShortName=Debian;ShortVersion=9 is also acceptable